### PR TITLE
Add status icons to manga info

### PR DESCRIPTION
**Pull request by Soitora**
One of my final changes with my vision of a better Manga page.
Ethically produced and approved by resident mogub:er.

We were a bit _think_ on the icon for `unknown_status`, I think it fits but could also be set to no icon.

I wondered if there was a way to set `width` to match `height` but couldn't solve it. I could set height to follow the container so it matches text no matter the size, but I couldn't then figure out how to make it a perfect square still.

Is the size in general good or too big? 18dp kinda matches closely to the text itself but that doesn't feel like standard practice doing it like that.

(For fork maintainers with additional statuses, it's of course stupid simple to add support for this, looking at you _Yo Bobby_)

### Images
| Previews | Previews  |
| ------- | ------- |
| Completed | Licensed |
| Ongoing | Unknown |

### CODE REVIEW

**Comment by CrepeTF** [comment]
TachiXD soontm

**Comment by arkon** [comment]
Shouldn't unknown be a question mark or something?

**Comment by CrepeTF** [comment]
> Shouldn't unknown be a question mark or something?

We tried that first, but it looked kinda bad, also for some reason Google just straight up doesn't have a normal question mark icon

File: app/src/main/res/layout-sw720dp/manga_info_header.xml [hunk 1]
```diff
@@ -95,6 +95,16 @@
...
             android:layout_height="wrap_content">
 
+            <ImageView
+                android:id="@+id/manga_status_icon"
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:layout_marginEnd="4dp"
+                android:background="@drawable/ic_status_unknown_24dp"
+                android:backgroundTint="?android:attr/textColorSecondary"
+                android:visibility="visible"
```
Comment (arkon) [inline app/src/main/res/layout-sw720dp/manga_info_header.xml:105]: Not needed?

File: app/src/main/res/layout-sw720dp/manga_info_header.xml [hunk 2]
```diff
@@ -95,6 +95,16 @@
...
+                tools:ignore="HardcodedText" />
```
Comment (arkon) [inline app/src/main/res/layout-sw720dp/manga_info_header.xml:106]: Not needed? If anything, should be `tools:ignore="ContentDescription"`.

File: app/src/main/res/layout-sw720dp/manga_info_header.xml [hunk 3]
```diff
@@ -95,6 +95,16 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content">
 
+            <ImageView
+                android:id="@+id/manga_status_icon"
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:layout_marginEnd="4dp"
+                android:background="@drawable/ic_status_unknown_24dp"
+                android:backgroundTint="?android:attr/textColorSecondary"
```
Comment (arkon) [inline app/src/main/res/layout-sw720dp/manga_info_header.xml:104]: `app:tint`?

File: app/src/main/res/layout-sw720dp/manga_info_header.xml [hunk 4]
```diff
@@ -95,6 +95,16 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content">
 
+            <ImageView
+                android:id="@+id/manga_status_icon"
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:layout_marginEnd="4dp"
+                android:background="@drawable/ic_status_unknown_24dp"
```
Comment (arkon) [inline app/src/main/res/layout-sw720dp/manga_info_header.xml:103]: `app:srcCompat`?

File: app/src/main/res/layout-sw720dp/manga_info_header.xml (hunk 1 above)
Comment (Soitora) [reply app/src/main/res/layout-sw720dp/manga_info_header.xml:105]: Ahh I forgot to remove it in tablet when i did my final quick commit

File: app/src/main/res/layout-sw720dp/manga_info_header.xml (hunk 4 above)
Comment (Soitora) [reply app/src/main/res/layout-sw720dp/manga_info_header.xml:103]: Didn't know this was a thing but I'll try it later

**Comment by Soitora** [comment]
> Shouldn't unknown be a question mark or something?

Didn't think it'd fit:

image

File: app/src/main/res/layout-sw720dp/manga_info_header.xml (hunk 2 above)
Comment (Soitora) [reply app/src/main/res/layout-sw720dp/manga_info_header.xml:106]: Same as the above, my I removed it in my final quick commit before sleep *and I forgot Tablet*

**Comment by Soitora** [comment]
Was told the icons is aligned but it felt 1dp more to the top, but what do I know. It's ready if there's no further comments.

**Pull request merged**